clientinfo.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DNWS
+{
+  class clientinfo : IPlugin

```

Comment:
1. You should follow the style guideline, the class name should be in the title case, i.e., ClientInfoPlugin.
2. You should follow the pattern, i.e., put "Plugin" as suffix to all plugin class.